test/retry.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+// TODO: browser connection timeouts will have different error messages, but we need to wait for them to occur in order
+// to find out what they are
+function isTimeout(e : Error) {
+    return e.message.includes("AWS_IO_TLS_NEGOTIATION_TIMEOUT") || e.message.includes("AWS_IO_SOCKET_TIMEOUT");
```

Comment:
Does this mean the retry function would not work for browser tests? While I saw it has been applied for all browser tests already.